docs/Developers/Writing_Good_Commits.rst

```diff
@@ -144,17 +144,18 @@ Example:
 
     Fixes https://github.com/coala/coala/issues/5861
 
--  Should use the ``Fixes`` keyword if your commit fixes a bug, or ``Closes``
-   if it adds a feature/enhancement.
+-  Should use the ``Fixes`` keyword if your commit fixes a bug, ``Closes``
+   if it adds a feature/enhancement, or ``Related to`` if the commit does
```

Comment:
I think It should be - ```Related to``` if the commit is related to some issue and doesn't close it.

Reply:
The description is exactly where your comment is, i.e: ``Related to`` if the commit does not close the issue.

Reply:
Yes this should be more descriptive as it's not understandable in one time.